logger/logger_writer.go

```diff
@@ -20,24 +19,53 @@ import (
 // same signature as printf.
 
 // #include <stdlib.h>
+// #ifdef _WIN32               /* for Windows builds, syslog functions do NOTHING */
```

Comment:
The change allows building/testing this repo's code in Windows. In other words, I can run `go test` for this repo.

I don't think I can build all of `kproxy`/`chf` in Windows. But haven't tried it yet.

Reply:
There is perhaps a slightly cleaner way to handle this.  Instead of using `#ifdef` in the CGO, perhaps we could move the CGO out of this file and create separate CGO files for windows/linux using Go's conditional compilation.
https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

The interesting thing about this approach is you can also stub out exported Go functions to be no-ops on Windows (no idea if that's useful here, but I've done this in the past)

Reply:
That's definitely worth looking in @cleroux! I've created an issue for now.